#!/usr/bin/env python
PACKAGE = "pico_zense_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("depth_confidence_threshold", int_t, 0,
        "Depth distance confidance (0% keeps all, 100% drops every depth point with <100% confidence", 0,  0, 100)
gen.add("depth_range", double_t, 0, "Optimise the camera for this depth range", 1.0, 0, 5.6)

exit(gen.generate(PACKAGE, "pico_zense_camera", "pico_zense_dcam710"))